packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -26,8 +26,10 @@
     "@salesforce/salesforcedx-utils-vscode": "43.14.0",
     "ajv": "^6.1.1",
     "eslint": "4.16.0",
-    "eslint-plugin-lwc": "0.4.0",
-    "lwc-language-server": "1.6.6",
+    "eslint-plugin-lwc":
+      "http://npm.lwcjs.org/eslint-plugin-lwc/-/eslint-plugin-lwc-0.4.0/643f5e18247ca8c0a60bf972f4d3795c97e8e6fe.tgz",
+    "lwc-language-server":
+      "http://npm.lwcjs.org/lwc-language-server/-/lwc-language-server-1.6.1/48af811b08f90d445c0a5185071129d29df73d39.tgz",
```

Comment:
We were using version 1.6.6 and this tarball is 1.6.1, any reason for not using 1.6.6 tarball ?

Reply:
Good catch. No this is a typo when I was copy-and-pasting. We should use 1.6.6. Let me fix this.